apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -19,6 +19,7 @@
 import com.codahale.metrics.MetricRegistry;
 import com.codahale.metrics.Timer;
 import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
```

Comment:
Do we need to use this over `Collectors.toList()`?

Reply:
Immutable Collections provide:
Less heap space: The space required to store a collection data is very less as compared with the traditional approach in earlier versions of java.
Faster access to data: As the overhead to store data and wrap into Collections.unmodifiable is reduced, now data access becomes faster.